Apps/W1/HybridGP/app/src/Migration/Customers/GPCustomerMigrator.codeunit.al

```diff
@@ -172,6 +176,86 @@ codeunit 4018 "GP Customer Migrator"
                 Sender.SetGeneralJournalLineExternalDocumentNo(CopyStr(MigrationGPCustTrans.DOCNUMBR, 1, 20) + '-' + CopyStr(MigrationGPCustTrans.GLDocNo, 1, 14));
             until MigrationGPCustTrans.Next() = 0;
     end;
+#pragma warning restore AA0207
+
+    local procedure CreateCustomerPostingGroupIfNeeded(var Sender: Codeunit "Customer Data Migration Facade"; RecordIdToMigrate: RecordId)
```

Comment:
Don't name the codeunit Sender, but Customer Data Migration. It is needed in the subscriber, but it should not be needed here.

Reply:
Updated for both the Customer and Vendor migrators. Thanks